AI Game Jam/Assets/Scripts/PickupItem.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickupItem : MonoBehaviour
+{
+    public bool inRange;
+    public GameObject item;
+
+    // Update is called once per frame
+    void Update()
+    {
+        PickUp();
+    }
+
+    void OnTriggerEnter(Collider other)
+    {
+        if(other.gameObject.tag == "Item") //if the collided object is the player
+        {
+            inRange = true; //set inRange to true
+            item = other.gameObject; //assign the item as the item for pickup
+            Debug.Log("In Range");
```

Comment:
These debug logs should be able to be removed now. just to tidy up the code.

Reply:
Done!